sdk/synapse/Azure.Analytics.Synapse.Artifacts/api/Azure.Analytics.Synapse.Artifacts.netstandard2.0.cs

```diff
@@ -998,6 +1159,7 @@ public partial class AzureBlobStorageLinkedService : Azure.Analytics.Synapse.Art
     {
         public AzureBlobStorageLinkedService() { }
         public Azure.Analytics.Synapse.Artifacts.Models.AzureKeyVaultSecretReference AccountKey { get { throw null; } set { } }
+        public string AccountKind { get { throw null; } set { } }
```

Comment:
Could this be an extensible enum?

Reply:
Possibly - we'd need to look at it in the context of the rest of the API.